core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/TuningEntryTrait.scala

```diff
@@ -81,12 +93,23 @@ trait TuningEntryTrait {
     if (isUnresolved()) {
       fillIfBlank.getOrElse(fillUnresolved.get)
     } else {
-      // It is possible the the propery was not tuned. However, we should not be in that case
+      // It is possible the property was not tuned. However, we should not be in that case
       // because by calling commit we must have copied the tuned from the original.
-      tunedValue.getOrElse(fillIfBlank.getOrElse(originalValue.getOrElse("[UNDEFINED]")))
+      val finalTunedValue = tunedValue.map(formatOutput)
+      val finalOriginalValue = originalValue.map(formatOutput)
+      finalTunedValue.orElse(fillIfBlank).orElse(finalOriginalValue)
+        .getOrElse("[UNDEFINED]")
     }
   }
 
+  /**
+   * Returns the original value as a string.
+   * @return the value of the property as a string.
+   */
+  def getOriginalValue: Option[String] = {
+    originalValue.map(formatOutput)
```

Comment:
Could have moved this to getter with caching to avoid calling `formatOutput()` multiple times, but keeping `getOriginalValue()` explicit felt cleaner.